docs/hooks.md

```diff
@@ -1,7 +1,10 @@
 ## Embedded hooks
 
-In addition to our [React Integration](./react-integration), Cozy Client comes with several hooks.
+In addition to our [React Integration](../react-integration), Cozy Client comes with several hooks.
 
-- [useAppLinkWithStoreFallback](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/hooks/useAppLinkWithStoreFallback.jsx): Returns the URL of an app if this app is installed. If not, returns the URL to the store to install it. 
+- [useAppLinkWithStoreFallback](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/hooks/useAppLinkWithStoreFallback.jsx): Returns the URL of an app if this app is installed. If not, returns the URL to the store to install it.
 - [useCapabilities](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/hooks/useCapabilities.jsx): Returns the [capabilities](https://docs.cozy.io/en/cozy-stack/settings/#get-settingscapabilities) for an instance
-- [useFetchShortcut](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/hooks/useFetchShortcut.jsx): Returns the data for a shortcut
\ No newline at end of file
+- [useFetchShortcut](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/hooks/useFetchShortcut.jsx): Returns the data for a shortcut
+- [useClient](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/hooks/useClient.js): Returns client of actual context
+- [useFetchJSON](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/hooks/useFetchJSON.js): Hook to use the generic fetchJSON method. Returns object with the same keys { data, fetchStatus, error } as useQuery
+- [useQuery](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/hooks/useQuery.js): Returns the queryState after fetching a queryDefinition
```

Comment:
Thanks for checking those links 👍 

I see a potential problem here : our docs are generated on https://docs.cozy.io and here the links are pointing to github.com. I think we should use relative links in those cases so that links work on docs.cozy.io as well as on github.com (see the React Integration) link.

Reply:
Ok we should have those docs in https://docs.cozy.io first, I think it's not already done. I'll check this out

Reply:
The docs in docs.cozy.io are pulled automatically from various repos, including this one. The file that we are discussing is the one that will get displayed on docs.cozy.io :)